bundle/app.go

```diff
@@ -47,7 +47,7 @@ func (b *Bundle) GetApp(id uint64) (*apistructs.ApplicationDTO, error) {
 	return &fetchResp.Data, nil
 }
 
-func (b *Bundle) GetMyApps(userid string, orgid uint64) (*apistructs.ApplicationListResponseData, error) {
+func (b *Bundle) GetMyApps(userid string, orgid, projectID uint64) (*apistructs.ApplicationListResponseData, error) {
```

Comment:
please keep the old func, and add a new func, unless, orchestrator will be affected.